backend/src/main/java/com/redhat/cloud/notifications/db/repositories/NotificationRepository.java

```diff
@@ -23,7 +23,7 @@ public class NotificationRepository {
     EntityManager entityManager;
 
     public List<NotificationHistory> getNotificationHistory(String orgId, UUID endpoint, boolean includeDetails, Query limiter) {
-        String query = "SELECT NEW NotificationHistory(nh.id, nh.invocationTime, nh.invocationResult, nh.endpoint, nh.created";
+        String query = "SELECT NEW NotificationHistory(nh.id, nh.invocationTime, nh.status, nh.endpoint, nh.created";
```

Comment:
Is the `invocationResult` no longer needed in the response of the REST endpoint that invokes this method?

Reply:
I was planning to remove the `invocationResult` column from the table and compute it from the `status` value - and then decided to leave it for a followup PR - I'll check if this was a remaining of that and revert the change.